File: modul-01/python/PRAK104-2310817310003-Mfaridsetiawan.py
HargasepatuA = int(input("Harga Sepatu A adalah: "))
HargasepatuB = int(input("Harga Sepatu B adalah: "))

HargasepatuA_diskon = 0.13
HargasepatuB_diskon = 0.21

print(f"Sepatu A mendapat diskon 13%% sehingga harga menjadi {HargasepatuA - (HargasepatuA * HargasepatuA_diskon):.0f}")
print(f"Sepatu B mendapat diskon 21%% sehingga harga menjadi {HargasepatuB - (HargasepatuB * HargasepatuB_diskon):.0f}")

File: modul-01/bhasa c/PRAK104-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

    int main(){

    int hrg_a;
    printf("harga sepatu a adalah:");
    scanf("%d", &hrg_a);
    int hrg_b;
    printf("harga sepatu b adalah:");
    scanf("%d", &hrg_b);

    float hrg_adiskon= 0.13;
    float hrg_bdiskon= 0.21;
    printf("Sepatu A mendapat diskon 13%% sehingga harganya menjadi: %.0f\n", hrg_a - (hrg_a * hrg_adiskon));
    printf("Sepatu B mendapat diskon 21%% sehingga harganya menjadi: %.0f\n", hrg_b - (hrg_b * hrg_bdiskon));
    return 0; 
    }